Makefile-docker

```diff
@@ -211,52 +190,57 @@ setup-codestyle:
 	$(PIP_COMMAND) install --progress-bar=off --no-deps --exists-action=w -r requirements/codestyle.txt
 
 .PHONY: lint-codestyle
-lint-codestyle:
+lint-codestyle: ## lint the code
 	flake8 src/ services/ tests/
 
 .PHONY: docs
-docs:
+docs: ## build the documentation
 	$(MAKE) -C docs html SPHINXOPTS='-nW'
 
 .PHONY: debug
-debug:
+debug: ## connect for debugging
 	supervisorctl fg olympia
 
 .PHONY: djshell
-djshell:
+djshell: ## connect to django shell
 	$(PYTHON_COMMAND) ./manage.py shell_plus
 
 .PHONY: dbshell
-dbshell:
+dbshell: ## connect to a database shell
 	$(PYTHON_COMMAND) ./manage.py dbshell
 
 .PHONY: initialize
-initialize: update_deps initialize_db update_assets populate_data
+initialize: update_deps initialize_db update_assets populate_data ## init the dependencies, the database, and assets
 
 .PHONY: reload-uwsgi
```

Comment:
It didn't have one before, but it'd be nice if this had a comment too so it shows up